linux-syscall/src/lib.rs

```diff
@@ -4,8 +4,17 @@
 #![no_std]
 #![deny(clippy::all)]
 
+// SysCall values generated with:
+//
+// ```
+// bindgen /usr/include/sys/syscall.h \
+//   | sed -rn 's|pub const SYS_([a-z][a-z0-9_]*): u32 = ([0-9]+);|\1 = \2,|p' \
+//   | tr [:lower:] [:upper:] \
+//   | sort
```

Comment:
There is no objective basis for this, but would you consider sorting on the integer value instead of the symbol name:

```
| sort -g -t "=" -k 2
```